src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -187,16 +187,26 @@ export class PassFinalisationPage extends PracticeableBasePageComponent {
   }
 
   getFormValidation(): { [key: string]: FormControl } {
+    const passCerticateFormat = new RegExp(/\D{1}\d{6}\W{1}/);
```

Comment:
Won't this accept symbols for the first character (anything that's not a digit)? Shouldn't you use `[A-Z]`?

Reply:
You're 100% right, fixed and pushing changes once I check out the comment above.

Reply:
Spoke with Andy around the last digit, this has also been amended.